Client/Frontend/Settings/Clearables.swift

```diff
@@ -126,24 +115,14 @@ class CacheClearable: Clearable {
             // Remove the basic cache.
             URLCache.shared.removeAllCachedResponses()
             
-            var err: Error?
-            for item in ["Caches", "Cookies", "WebKit"] {
-                do {
-                    try deleteLibraryFolderContents(item, validateClearedExceptFor: ["Snapshots"])
-                } catch {
-                    err = error
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes(), modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
+                // Clear image cache
+                ImageCache.shared.clear()
+                
+                // Leave the cache off in the error cases above
```

Comment:
This says `in the error cases above` but we removed errors with the new implementation, do we still need this? Maybe the comments is just outdated, might require some research

Reply:
Removed the comments as they are outdated